oneview_redfish_toolkit/util.py

```diff
@@ -284,3 +286,66 @@ def get_oneview_client():
         # if faild abort
         except Exception:
             raise
+
+
+def get_ip():
+    """Tries to detect default route IP Address"""
+    s = socket.socket(type=socket.SOCK_DGRAM)
+    try:
+        s.connect(("8.8.8.8", 1))
+        IP = s.getsockname()[0]
+    except Exception as e:
+        logging.exception(e)
+        IP = "127.0.0.1"
+    finally:
+        s.close()
+    return IP
+
+
+def generate_certificate(dir_name, file_name, key_length, key_type="rsa"):
+    """Create self-signed cert and key files
+
+        Args:
+            dir_name: name of the directory to store the files
+            file_name: name of the files that will be created. It will append
+                .crt to certificate file and .key to key file
+            key_length: key length in bits
+            key_type: cryto type: RSA or DSA; defaults to RSA
+        Returns:
+            Nothing
+        Exceptions:
+            Raise exceptions on error
+    """
+
+    config = globals()['config']
```

Comment:
What happened here? I thought `import util.config as config` was supposed to work to avoid this kind of thing...

Reply:
import util.config on the util file. This is very strange , never imported a file in the same file and really didn't know i could do that. Does it work? Does it make sense? I look way too strange to my eyes.